budoux/main.py

```diff
@@ -122,13 +122,21 @@ def parse_args(test: ArgList = None) -> argparse.Namespace:
       type=check_lang,
       help="language of custom model",
   )
+  parser.add_argument(
+      "-D",
+      "--phrase-delimiter",
```

Comment:
Noticed you filed #793. Please let me know if you prefer other option names.

Reply:
Thank you for this PR! I prefer `--sep` and `-s` because it's shorter.

Reply:
Ack, done.